### PR TITLE
Help River

### PR DESCRIPTION
**Pull request by jeigsti777**
River I'm not sure what to do next. @HotShot3150

### CODE REVIEW

File: models/Question.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+let Question = new Schema({
+    id: {
+        type: number
+    },
```
Comment (HotShot3150) [inline models/Question.js]: When creating the schema mongoose will auto create the id field so this shouldn't be needed.

File: models/Question.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    answer3: {
+        type: String
+    },
+    answer4: {
+        type: String
+    },
+    answer5: {
+        type: String
+    }
```
Comment (HotShot3150) [inline models/Question.js]: This being the question model we do not need to have a set number of answers predefined.

Proposed Schema
QUESTION:
description : String
assessment_id : number
answership : [need to research how to setup this field] (has_many relationship)
updated_at : {type: Date, default: Date.now() }

We can discuss this further in an upcoming meeting when we go over all our schemas for the assessment.

File: routes/question.route.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const app = express();
+const questionRoute = express.Router();
+
+//Question model
+let Question = require('../models/Question');
+
+//Get All Questions
+questionRoute.route('/').get((req, res) => {
```
Comment (HotShot3150) [inline routes/question.route.js:9]: I'm not familiar with this syntax for routes. Lets meet up on this and see which routing will work best for us. Take a look at my User-Schema pull request to see what I used for the routes. Might not matter which we use.

**Review by HotShot3150 — CHANGES_REQUESTED** [review]

**Comment by jeigsti777** [comment]
@HotShot3150 I updated the question model. We can talk about the routing if you have another path you want to take.

**Pull request merged**